### PR TITLE
V15 Added Tiptap test helper

### PR DESCRIPTION
**Pull request by nhudinh0309**

### CODE REVIEW

File: lib/helpers/DataTypeUiHelper.ts [hunk 1]
```diff
@@ -630,7 +642,7 @@ export class DataTypeUiHelper extends UiBaseLocators {
   }
 
   // Richtext Editor
-  async pickTheToolbarOptionByValue(values) {
+  async clickTheToolbarOptionByValue(values) {
```
Comment (andr317c) [inline lib/helpers/DataTypeUiHelper.ts:645]:
```suggestion
  async clickToolbarOptionByValue(values) {
```

File: lib/helpers/DataTypeUiHelper.ts [hunk 2]
```diff
@@ -999,4 +1012,47 @@ export class DataTypeUiHelper extends UiBaseLocators {
...
+
+  async clickAddRowToolbarButton() {
+    await this.addRowToolbarBtn.click();
+  }
+
+  async clickAddGroupToolbarButton() {
+    await this.addGroupToolbarBtn.click();
+  }
+
+  async clickExtensionItemHaveName(name: string) {
```
Comment (andr317c) [inline lib/helpers/DataTypeUiHelper.ts:1037]:
```suggestion
  async clickExtensionItemWithName(name: string) {
```

File: lib/helpers/DataTypeUiHelper.ts [hunk 3]
```diff
@@ -999,4 +1012,47 @@ export class DataTypeUiHelper extends UiBaseLocators {
...
+    await this.tiptapExtensionsConfiguration.locator('uui-checkbox[label="' + name + '"]').click();
+  }
+
+  async addToolbarItemToGroup(itemName: string, rowIndex: number = 0) {
```
Comment (andr317c) [inline lib/helpers/DataTypeUiHelper.ts:1041]: Should we specify the group Index for this?

**Review by andr317c — CHANGES_REQUESTED** [review]
Had a few comments

File: lib/helpers/DataTypeUiHelper.ts (hunk 3 above)
Comment (nhudinh0309) [reply lib/helpers/DataTypeUiHelper.ts:1041]: Removed it as it is unnecessary now.

**Review by andr317c — APPROVED** [review]
LGTM

**Pull request merged**